src/client/src/reducers/index.ts

```diff
@@ -21,5 +23,29 @@ const rootReducer = combineReducers({
   dependencyInfo
 });
 
-export type AppState = ReturnType<typeof rootReducer>;
+export type AppState = ReturnType<typeof appReducer>;
+
+const rootReducer = (state: AppState | undefined, action: RootAction) => {
+  let passedState: any;
```

Comment:
Anyone have any ideas on how to handle the dynamically changing type of the appReducer here? I don't want to use any really.

Reply:
I think you'd have to caste the state into its correct type once you figure out what it is, if that's possible.

Reply:
Yeah I don't think its worth the effort here really, adds so much overhead to the developer, if they modify the state.